R/class-Link.R

```diff
@@ -0,0 +1,87 @@
+
+
+check_link <- function(object) {
+
+  errors <- character()
+
+  # Link color must be a string or number
+  if (!is.null(object@color) & !is.character(object@color) & !is.numeric(object@color)) {
+    errors <- c(errors, "Link color must be a string or number that represents a color or can be mapped to a color.")
```

Comment:
wait a second, why not date, too? Food for future thought. Actually that'd probably be handeled in whatever setLinkColors method accepts date data.

Reply:
im gonna punt on this until we have a case in hand.